lib/components/form/advanced-settings-panel.tsx

```diff
@@ -99,6 +99,24 @@ const ButtonContainer = styled.div`
   margin-top: 2em;
 `
 
+const DtSelectorContainer = styled.div`
+  padding: 1em;
```

Comment:
I think we should add some left right padding in here somewhere for the buttons. The text is squished at widths 992-1150ish px
<img width="318" alt="image" src="https://github.com/user-attachments/assets/c55cdb95-8965-4437-b298-d4ab8ebc1228">

Reply:
Good call! I removed the padding on these actually since the text is centered, now when the buttons resize it should be way more graceful. Fixed in [d4215c5](https://github.com/opentripplanner/otp-react-redux/pull/1248/commits/d4215c58eff30662b2fd78761ec1b3f69bb221fa)